src/features/events_list/atoms/autoSelectEvent.ts

```diff
@@ -23,10 +21,20 @@ export const autoSelectEvent = createAtom(
       ) {
         const currentEvent = getUnlistedState(currentEventAtom);
         const currentEventNotInNewList =
-          eventListResource.data.findIndex(
-            (e) => e.eventId === currentEvent?.id,
-          ) === -1;
-        if (currentEventNotInNewList) {
+          eventListResource.data.findIndex((e) => e.eventId === currentEvent?.id) === -1;
+
+        if (currentEvent?.id && currentEventNotInNewList) {
+          // This case happens when call for event by provided url id didn't return event
+          schedule((dispatch) =>
+            dispatch(
+              currentNotificationAtom.showNotification(
+                'warning',
+                { title: i18n.t('event_list.no_event_in_feed') },
+                5,
+              ),
+            ),
+          );
+        } else if (currentEventNotInNewList) {
```

Comment:
i would suggest to get rid of else if logic 
like here (feels simplier)
```
      if (currentEventNotInNewList) {
          if (currentEvent?.id) {
            // This case happens when call for event by provided url id didn't return event
            schedule((dispatch) =>
              dispatch(
                currentNotificationAtom.showNotification(
                  'warning',
                  { title: i18n.t('event_list.no_event_in_feed') },
                  5,
                ),
              ),
            );
          } else {
            const firstEventInList = eventListResource.data[0];
            schedule((dispatch) => {
              dispatch([
                scheduledAutoSelect.setFalse(),
                scheduledAutoFocus.setTrue(),
                currentEventAtom.setCurrentEventId(firstEventInList.eventId),
              ]);
            });
          }
        }

Reply:
for me it's one more nesting\intendation level

Reply:
I did compromise